.github/workflows/examples-bouffalolab.yaml

```diff
@@ -52,21 +108,22 @@ jobs:
           GH_CONTEXT: ${{ toJson(github) }}
         run: scripts/tools/memory/gh_sizes_environment.py "${GH_CONTEXT}"
 
+
       - name: Build example BL702 Lighting App
         timeout-minutes: 30
         run: |
           ./scripts/run_in_build_env.sh \
-            "./scripts/build/build_examples.py --target bouffalolab-bl706-iot-dvk-light build"
+            "./scripts/build/build_examples.py --target bouffalolab-xt-zb6-devkit-light build"
           .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py bl702 bl702 lighting-app \
-            out/bouffalolab-bl706-iot-dvk-light/chip-bl702-lighting-example.out /tmp/bloat_reports/
+            out/bouffalolab-xt-zb6-devkit-light/chip-bl702-lighting-example.out /tmp/bloat_reports/
 
       - name: Build example BL702 Lighting App with RPCs
         timeout-minutes: 30
         run: |
           ./scripts/run_in_build_env.sh \
-            "./scripts/build/build_examples.py --target bouffalolab-bl706-iot-dvk-light-rpc build"
+            "./scripts/build/build_examples.py --target bouffalolab-xt-zb6-devkit-light-115200-rpc build"
```

Comment:
If this changes, then integrations/cloudbuild/build-all.yaml should probably also change.

Reply:
build-all.yaml updated